src/btree/bt_curnext.c

```diff
@@ -660,14 +649,12 @@ __wt_btcur_next(WT_CURSOR_BTREE *cbt, bool truncating)
 		if (F_ISSET(cbt, WT_CBT_ITERATE_APPEND)) {
 			switch (page->type) {
 			case WT_PAGE_COL_FIX:
-				ret = __cursor_fix_append_next(
-				    cbt, newpage, restart);
+				ret = __cursor_fix_append_next(cbt, newpage, restart);
 				break;
 			case WT_PAGE_COL_VAR:
-				ret = __cursor_var_append_next(
-				    cbt, newpage, restart);
+				ret = __cursor_var_append_next(cbt, newpage, restart);
 				break;
-			WT_ILLEGAL_VALUE_ERR(session, page->type);
+				WT_ILLEGAL_VALUE_ERR(session, page->type);
```

Comment:
Known issue. This just looks like a function call so it's understandable that it gets indented under the `case` statement above.

I think the solution is to just remove this macro entirely since it doesn't provide much value.

Reply:
@tetsuo-cpp do you mean remove the macro and replace uses with explicit `default:` labels?  There is value in this kind of error checking, even if the macro causes problems.

Reply:
Yeah sorry, I should have clarified. I was suggesting to remove the macro and just write the contents inline. It won't be much more verbose since it just contains this:

```
#define WT_ILLEGAL_VALUE(session, v)                                    \
        default:                                                        \
                return (__wt_illegal_value(session, v))
```